src/main/java/org/dcsa/ovs/notifications/config/ApplicationConfig.java

```diff
@@ -0,0 +1,16 @@
+package org.dcsa.ovs.notifications.config;
+
+import org.dcsa.core.repository.ExtendedRepositoryImpl;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@Configuration
+@ComponentScan("org.dcsa")
+@EnableR2dbcRepositories(
+    basePackages = {"org.dcsa"},
+    repositoryBaseClass = ExtendedRepositoryImpl.class)
+
+@EnableScheduling
```

Comment:
Do we have any scheduling in Notifications?

Reply:
I dont know what it is supposed to do. Copied from TNT.

Reply:
We will need `@EnableScheduling` if subscription come in here too.

Reply:
Yes - I agree. My impression was that pending_events "stayed" in OVS and only CRUD operations was moved.
In accordance with Niels T - everything related to Subscriptions and Events should go here - so we keep @enableScheduling :-)